impl/src/main/java/com/okta/sdk/impl/api/OAuth2ApiClientCredentialsResolver.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.api;
+
+import com.okta.sdk.authc.credentials.ClientCredentials;
+
+/**
+ * @since 1.6.0
+ */
+public class OAuth2ApiClientCredentialsResolver implements ClientCredentialsResolver {
```

Comment:
It looks like this could be replaced by using the `DefaultClientCredentialsResolver`